indexing-service/src/main/java/io/druid/indexing/common/actions/LocalTaskActionClient.java

```diff
@@ -59,6 +59,13 @@ public LocalTaskActionClient(Task task, TaskStorage storage, TaskActionToolbox t
       }
     }
 
-    return taskAction.perform(task, toolbox);
+    RetType rv = taskAction.perform(task, toolbox);
+    log.info(
```

Comment:
wasn't sure if its useful as a metric or not, 
can promote it to metric if you find it useful ?